pkg/services/dashboards/folder_service.go

```diff
@@ -32,7 +33,7 @@ var NewFolderService = func(orgID int64, user *models.SignedInUser, store dashbo
 	}
 }
 
-func (dr *dashboardServiceImpl) GetFolders(limit int64) ([]*models.Folder, error) {
+func (dr *dashboardServiceImpl) GetFolders(ctx context.Context, limit int64) ([]*models.Folder, error) {
```

Comment:
This uses `bus.Dispatch` for search.Query, should we have a separate PR for that or pull that into this?

Reply:
Separate is the idea. Trying to limit the amount of changes to smaller PRs that's easier to review and merge #36734